key-export-client/src/lib.rs

```diff
@@ -8,12 +8,22 @@
 #![forbid(missing_docs)]
 #![no_std]
 
+// Because JsValue is not suported in non-wasm32 architectures,
+// this code is compiled to return a different error type and
+// request type in wasm32 and non-wasm32 architectures.
+#[cfg(not(target_arch = "wasm32"))]
+mod utils_non_wasm32;
+#[cfg(target_arch = "wasm32")]
+mod utils_wasm32;
+
+#[cfg(not(target_arch = "wasm32"))]
+use utils_non_wasm32 as utils;
+#[cfg(target_arch = "wasm32")]
+use utils_wasm32 as utils;
```

Comment:
There's a more Rust-ish way to do that. (assuming we've renamed `utils` into `types`) You can create a folder `types` move `types_wasm32.rs` into `types/wasm32.rs` and `types_non_wasm32.rs` into `types/others.rs`.

Then you can write:
```suggestion
#[cfg_attr(target_arch = "wasm32", path = "types/wasm32.rs")]
#[cfg_attr(not(target_arch = "wasm32"), path = "types/others.rs")]
mod types;
```

Reply:
Oh yes, much better!